samples/secret/secret.yaml

```diff
@@ -51,6 +51,20 @@ arrays:
     # Default Value: None
     nasName: "nas-server"
 
+    # nfsAcls: enables setting permissions on NFS mount directory
+    # This value will be used if a storage class does not have the NFS ACL (nfsAcls) parameter specified
+    # Permissions can be specified in two formats:
+    #   1) Unix mode (NFSv3)
+    #   2) NFSv4 acl (NFSv4)
+    # Allowed values:
+    #   1) Unix mode: valid octal mode number
+    #      Examples: "0777", "777", "0755"
+    #   2) NFSv4 acls: valid NFSv4 acls, seperated by comma
```

Comment:
Please mention that only NFSV4 ACLs are supported

Reply:
done